Project/structure.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="widtfh=device-width, initial-scale=1">
+        <title>BikeTourism</title>
+    </head>
+
+    <body>
+        <header>
+            <nav>
+                <ul>
+                    <li>main</li>
+                    <li>about</li>
+                    <li>contact</li>
+                </ul>
+            </nav>
+        </header>
+
+        <section class="main-content">
+            <section class="cities-list">
+                <h1>title</h1>
+                <p>short description</p>
+                <ul>
+                    <li></li>
+                    <li></li>
+                    <li></li>
+                    <li></li>
+                    <li></li>                                                                                                
+                </ul>
+            </section>
+
+            <section class="map">
+                <header class="map-header"></header>
+                <div class="map-container"></div>
+            </section>
+
+        </section>
+
+        <footer>
+            <p>
+                date info contact
+            </p>
+        </footer>
+    </body>
+
+</html>
```

Comment:
It's a standard to have a newline character at the end of line - https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline